src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -58,9 +58,25 @@ namespace
 
             m_gi = true;
             m_caustics = false;
-            m_bounces = 3;
+            m_bounces = 8;
+            m_dbounces = 3;
+            m_dbounces_set = false;
```

Comment:
And probably `m_diffuse_bounces_enabled` here.

Reply:
ok, no problem